test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Build.AspNetCore3x/BulkOperation/EFTests/BulkOperationPatchHandlersEF.cs

```diff
@@ -45,6 +45,7 @@ protected override void OnModelCreating(Microsoft.EntityFrameworkCore.ModelBuild
             modelBuilder.Entity<Friend>().HasKey(c => c.Id);
 
             modelBuilder.Entity<NewOrder>().Ignore(c => c.Container);
+            modelBuilder.Entity<NewOrder>().Ignore(c => c.InstanceAnnotations);
```

Comment:
why are we ignoring this?

Reply:
We don't need `InstanceAnnotations` to be mapped by EF Core